Src/BootCamp.Chapter/ShuffleDemo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal static class ShuffleDemo
+    {
+        internal static void Demo()
+        {
+            var shuffledList = ShuffleColllectionExtension(new List<int> { 1, 2, 3, 4, 5 });
```

Comment:
This is a demo for static method, but we asked you to demo extension methods. So it's completely wrong. Also, why do you pass a new list?

Reply:
You should shuffle the list that you created. Not pass it.

Reply:
oke, will work on that too